conductr_cli/bndl_main.py

```diff
@@ -364,6 +369,19 @@ def add_conf_arguments(parser):
                              'Example: bndl --volume my-vol=/data --component web',
                         metavar='NAME=MOUNT-POINT')
 
+    parser.add_argument('--with-defaults',
+                        default=None,
```

Comment:
No, I think the way you have it makes sense. I was thinking moving it to https://github.com/markusjura/conductr-cli/blob/bndl-defaults/conductr_cli/bndl_main.py#L373 but that clearly doesn't work upon further thought because we don't want it enabled for `bundle` and `configuration` formats. So -- disregard :)